README.md

```diff
@@ -88,17 +88,17 @@ In order to run Airkeeper with sample configuration files locally, you will need
 
 1. Open a new terminal and navigate to the root directory of the Airnode repo.
 2. Run `yarn run bootstrap && yarn build` to build the project.
-3. Add the config.json and secrets.env files to the packages/airnode-node/config directory.
+3. Add the `config.json` and `secrets.env` files to the packages/airnode-node/config directory.
 4. Run `yarn run dev:background` to start a local ethereum node and a sample REST API.
 5. Run `yarn run dev:eth-deploy` to deploy and configure the required contracts.
 6. Switch to the Airkeeper root directory and run `npm install`
 7. Finally run `npm run sls:invoke-local` to invoke the updateBeacon function.
 
 ### Running Airkeeper on AWS Lambda
 
-Airkeeper is meant to be deployed to AWS lambda service and for this you will need to setup your AWS credentials. This credentials should be added to the .env file in the root directory of the Airkeeper repo. Then you can use the export-env.sh script to load them into the environment.
+Airkeeper is meant to be deployed to AWS lambda service and for this you will need to setup your AWS credentials. These credentials should be added to the `aws.env` file in the root directory of the Airkeeper repo. Then you can use the export-env.sh script to load them into the environment.
 
```

Comment:
Is `in the root directory` above correct? Bullet #1 below refers to `aws.env` in `config/`

Reply:
Good catch. It seems I forgot to update this here when I moved and renamed the file.